infra/minio/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM minio/minio:RELEASE.2020-03-25T07-03-04Z
+CMD ["minio", "server", "/data"]
```

Comment:
Should we bind mount this?

Reply:
We don't have anything to bind mount it to right now.